### ISSUE TITLE
No Module Named Tensorflow

### ISSUE THREAD

**Issue by bw4sz** [conda-forge/deepforest-feedstock#2]
On a conda python 3.7 clean OSX environment. Everything installs, but there is no tensorflow.

```
MacBook-Pro:Figures ben$ conda create --name test python=3.7
WARNING: A conda environment already exists at '/Users/ben/miniconda3/envs/test'
Remove existing environment (y/[n])? y

Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /Users/ben/miniconda3/envs/test

  added / updated specs:
    - python=3.7


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    setuptools-47.3.1          |   py37hc8dfbb8_0         634 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         634 KB

The following NEW packages will be INSTALLED:

  ca-certificates    conda-forge/osx-64::ca-certificates-2020.4.5.2-hecda079_0
  certifi            conda-forge/osx-64::certifi-2020.4.5.2-py37hc8dfbb8_0
  libcxx             conda-forge/osx-64::libcxx-10.0.0-h1af66ff_2
  libffi             conda-forge/osx-64::libffi-3.2.1-h4a8c4bd_1007
  ncurses            conda-forge/osx-64::ncurses-6.1-h0a44026_1002
  openssl            conda-forge/osx-64::openssl-1.1.1g-h0b31af3_0
  pip                conda-forge/noarch::pip-20.1.1-py_1
  python             conda-forge/osx-64::python-3.7.6-cpython_h1fd5dd1_6
  python_abi         conda-forge/osx-64::python_abi-3.7-1_cp37m
  readline           conda-forge/osx-64::readline-8.0-hcfe32e1_0
  setuptools         conda-forge/osx-64::setuptools-47.3.1-py37hc8dfbb8_0
  sqlite             conda-forge/osx-64::sqlite-3.30.1-h93121df_0
  tk                 conda-forge/osx-64::tk-8.6.10-hbbe82c9_0
  wheel              conda-forge/noarch::wheel-0.34.2-py_1
  xz                 conda-forge/osx-64::xz-5.2.5-h0b31af3_0
  zlib               conda-forge/osx-64::zlib-1.2.11-h0b31af3_1006


Proceed ([y]/n)? y


Downloading and Extracting Packages
setuptools-47.3.1    | 634 KB    | ################################################################################################################################################################# | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate test
#
# To deactivate an active environment, use
#
#     $ conda deactivate

MacBook-Pro:Figures ben$ conda activate test
(test) MacBook-Pro:Figures ben$ python
Python 3.7.6 | packaged by conda-forge | (default, Jun  1 2020, 18:33:30)
[Clang 9.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> quit()
(test) MacBook-Pro:Figures ben$ conda install deepforest -c conda-forge
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /Users/ben/miniconda3/envs/test

  added / updated specs:
    - deepforest


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    cython-0.29.20             |   py37h570ac47_0         2.0 MB  conda-forge
    deepforest-0.2.19          |   py37h570ac47_0         8.8 MB  conda-forge
    gdal-3.0.4                 |   py37h08e9697_6         1.4 MB  conda-forge
    keyring-21.2.1             |   py37hc8dfbb8_0          46 KB  conda-forge
    libgdal-3.0.4              |       ha92c1e4_6        17.2 MB  conda-forge
    numpy-1.18.5               |   py37h7687784_0         5.0 MB  conda-forge
    numpydoc-1.0.0             |     pyh9f0ad1d_0          41 KB  conda-forge
    pandas-1.0.5               |   py37h94625e5_0         9.6 MB  conda-forge
    rasterstats-0.14.0         |             py_0          16 KB  conda-forge
    recommonmark-0.6.0         |             py_0          13 KB  conda-forge
    requests-2.24.0            |     pyh9f0ad1d_0          47 KB  conda-forge
    simplejson-3.17.0          |   py37h9bfed18_1         100 KB  conda-forge
    sphinx-3.1.1               |             py_0         1.4 MB  conda-forge
    sphinx_rtd_theme-0.5.0     |     pyh9f0ad1d_0         2.7 MB  conda-forge
    twine-3.1.1                |           py37_0          47 KB  conda-forge
    xmltodict-0.12.0           |             py_0          11 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        48.3 MB

The following NEW packages will be INSTALLED:

  affine             conda-forge/noarch::affine-2.3.0-py_0
  alabaster          conda-forge/noarch::alabaster-0.7.12-py_0
  attrs              conda-forge/noarch::attrs-19.3.0-py_0
  babel              conda-forge/noarch::babel-2.8.0-py_0
  bleach             conda-forge/noarch::bleach-3.1.5-pyh9f0ad1d_0
  boost-cpp          conda-forge/osx-64::boost-cpp-1.72.0-hdf9ef73_0
  brotlipy           conda-forge/osx-64::brotlipy-0.7.0-py37h9bfed18_1000
  bzip2              conda-forge/osx-64::bzip2-1.0.8-h0b31af3_2
  cairo              conda-forge/osx-64::cairo-1.16.0-hec6a9b0_1003
  cffi               conda-forge/osx-64::cffi-1.14.0-py37h356ff06_0
  cfitsio            conda-forge/osx-64::cfitsio-3.470-hddc9c06_5
  chardet            conda-forge/osx-64::chardet-3.0.4-py37hc8dfbb8_1006
  click              conda-forge/noarch::click-7.1.2-pyh9f0ad1d_0
  click-plugins      conda-forge/noarch::click-plugins-1.1.1-py_0
  cligj              conda-forge/noarch::cligj-0.5.0-py_0
  cmarkgfm           conda-forge/osx-64::cmarkgfm-0.4.2-py37h0b31af3_2
  commonmark         conda-forge/noarch::commonmark-0.9.1-py_0
  cryptography       conda-forge/osx-64::cryptography-2.9.2-py37he655712_0
  curl               conda-forge/osx-64::curl-7.69.1-h2d98d24_0
  cycler             conda-forge/noarch::cycler-0.10.0-py_2
  cython             conda-forge/osx-64::cython-0.29.20-py37h570ac47_0
  deepforest         conda-forge/osx-64::deepforest-0.2.19-py37h570ac47_0
  docutils           conda-forge/osx-64::docutils-0.16-py37hc8dfbb8_1
  entrypoints        conda-forge/osx-64::entrypoints-0.3-py37hc8dfbb8_1001
  expat              conda-forge/osx-64::expat-2.2.9-h4a8c4bd_2
  fiona              conda-forge/osx-64::fiona-1.8.13-py37h9c7205d_1
  fontconfig         conda-forge/osx-64::fontconfig-2.13.1-h6b1039f_1001
  freetype           conda-forge/osx-64::freetype-2.10.2-h8da9a1a_0
  freexl             conda-forge/osx-64::freexl-1.0.5-h1de35cc_1002
  future             conda-forge/osx-64::future-0.18.2-py37hc8dfbb8_1
  gdal               conda-forge/osx-64::gdal-3.0.4-py37h08e9697_6
  geos               conda-forge/osx-64::geos-3.8.1-h4a8c4bd_0
  geotiff            conda-forge/osx-64::geotiff-1.5.1-hd8796ba_10
  gettext            conda-forge/osx-64::gettext-0.19.8.1-h46ab8bc_1002
  giflib             conda-forge/osx-64::giflib-5.2.1-h0b31af3_2
  glib               conda-forge/osx-64::glib-2.64.3-h577aef8_0
  h5py               conda-forge/osx-64::h5py-2.10.0-nompi_py37h106b333_102
  hdf4               conda-forge/osx-64::hdf4-4.2.13-h84186c3_1003
  hdf5               conda-forge/osx-64::hdf5-1.10.5-nompi_h3e39495_1104
  icu                conda-forge/osx-64::icu-64.2-h6de7cb9_1
  idna               conda-forge/noarch::idna-2.9-py_1
  imagesize          conda-forge/noarch::imagesize-1.2.0-py_0
  importlib-metadata conda-forge/osx-64::importlib-metadata-1.6.1-py37hc8dfbb8_0
  importlib_metadata conda-forge/noarch::importlib_metadata-1.6.1-0
  jinja2             conda-forge/noarch::jinja2-2.11.2-pyh9f0ad1d_0
  jpeg               conda-forge/osx-64::jpeg-9d-h0b31af3_0
  json-c             conda-forge/osx-64::json-c-0.13.1-h575e443_1002
  kealib             conda-forge/osx-64::kealib-1.4.13-h2324030_0
  keyring            conda-forge/osx-64::keyring-21.2.1-py37hc8dfbb8_0
  kiwisolver         conda-forge/osx-64::kiwisolver-1.2.0-py37ha1cc60f_0
  krb5               conda-forge/osx-64::krb5-1.17.1-h1752a42_0
  libblas            conda-forge/osx-64::libblas-3.8.0-16_openblas
  libcblas           conda-forge/osx-64::libcblas-3.8.0-16_openblas
  libcurl            conda-forge/osx-64::libcurl-7.69.1-hc0b9707_0
  libdap4            conda-forge/osx-64::libdap4-3.20.6-h8c15375_0
  libedit            conda-forge/osx-64::libedit-3.1.20191231-hed1e85f_0
  libgdal            conda-forge/osx-64::libgdal-3.0.4-ha92c1e4_6
  libgfortran        conda-forge/osx-64::libgfortran-4.0.0-2
  libiconv           conda-forge/osx-64::libiconv-1.15-h0b31af3_1006
  libkml             conda-forge/osx-64::libkml-1.3.0-h169b8f9_1011
  liblapack          conda-forge/osx-64::liblapack-3.8.0-16_openblas
  libnetcdf          conda-forge/osx-64::libnetcdf-4.7.4-nompi_he461dc0_101
  libopenblas        conda-forge/osx-64::libopenblas-0.3.9-h3d69b6c_0
  libpng             conda-forge/osx-64::libpng-1.6.37-hbbe82c9_1
  libpq              conda-forge/osx-64::libpq-12.2-h489d428_1
  libspatialite      conda-forge/osx-64::libspatialite-4.3.0a-h658e6c1_1038
  libssh2            conda-forge/osx-64::libssh2-1.9.0-h39bdce6_2
  libtiff            conda-forge/osx-64::libtiff-4.1.0-h2ae36a8_6
  libwebp-base       conda-forge/osx-64::libwebp-base-1.1.0-h0b31af3_3
  libxml2            conda-forge/osx-64::libxml2-2.9.10-h53d96d6_0
  llvm-openmp        conda-forge/osx-64::llvm-openmp-10.0.0-h28b9765_0
  lz4-c              conda-forge/osx-64::lz4-c-1.9.2-h4a8c4bd_1
  markupsafe         conda-forge/osx-64::markupsafe-1.1.1-py37h9bfed18_1
  matplotlib-base    conda-forge/osx-64::matplotlib-base-3.2.1-py37hddda452_0
  more-itertools     conda-forge/noarch::more-itertools-8.4.0-py_0
  munch              conda-forge/noarch::munch-2.5.0-py_0
  numpy              conda-forge/osx-64::numpy-1.18.5-py37h7687784_0
  numpydoc           conda-forge/noarch::numpydoc-1.0.0-pyh9f0ad1d_0
  olefile            conda-forge/noarch::olefile-0.46-py_0
  openjpeg           conda-forge/osx-64::openjpeg-2.3.1-h254dc36_3
  packaging          conda-forge/noarch::packaging-20.4-pyh9f0ad1d_0
  pandas             conda-forge/osx-64::pandas-1.0.5-py37h94625e5_0
  pcre               conda-forge/osx-64::pcre-8.44-h4a8c4bd_0
  pillow             conda-forge/osx-64::pillow-7.1.2-py37hfd78ece_0
  pixman             conda-forge/osx-64::pixman-0.38.0-h01d97ff_1003
  pkginfo            conda-forge/noarch::pkginfo-1.5.0.1-py_0
  pluggy             conda-forge/osx-64::pluggy-0.13.1-py37hc8dfbb8_1
  poppler            conda-forge/osx-64::poppler-0.67.0-h16886b5_8
  poppler-data       conda-forge/noarch::poppler-data-0.4.9-1
  postgresql         conda-forge/osx-64::postgresql-12.2-h62ab893_1
  proj               conda-forge/osx-64::proj-7.0.0-hbce2d5c_4
  psutil             conda-forge/osx-64::psutil-5.7.0-py37h9bfed18_1
  py                 conda-forge/noarch::py-1.8.2-pyh9f0ad1d_0
  pycparser          conda-forge/noarch::pycparser-2.20-py_0
  pygments           conda-forge/noarch::pygments-2.6.1-py_0
  pyopenssl          conda-forge/noarch::pyopenssl-19.1.0-py_1
  pyparsing          conda-forge/noarch::pyparsing-2.4.7-pyh9f0ad1d_0
  pysocks            conda-forge/osx-64::pysocks-1.7.1-py37hc8dfbb8_1
  pytest             conda-forge/osx-64::pytest-5.4.3-py37hc8dfbb8_0
  python-dateutil    conda-forge/noarch::python-dateutil-2.8.1-py_0
  pytz               conda-forge/noarch::pytz-2020.1-pyh9f0ad1d_0
  pyyaml             conda-forge/osx-64::pyyaml-5.3.1-py37h9bfed18_0
  rasterio           conda-forge/osx-64::rasterio-1.1.5-py37h9c7205d_0
  rasterstats        conda-forge/noarch::rasterstats-0.14.0-py_0
  readme_renderer    conda-forge/noarch::readme_renderer-24.0-py_0
  recommonmark       conda-forge/noarch::recommonmark-0.6.0-py_0
  requests           conda-forge/noarch::requests-2.24.0-pyh9f0ad1d_0
  requests-toolbelt  conda-forge/noarch::requests-toolbelt-0.9.1-py_0
  shapely            conda-forge/osx-64::shapely-1.7.0-py37hfcf0db4_3
  simplejson         conda-forge/osx-64::simplejson-3.17.0-py37h9bfed18_1
  six                conda-forge/noarch::six-1.15.0-pyh9f0ad1d_0
  snowballstemmer    conda-forge/noarch::snowballstemmer-2.0.0-py_0
  snuggs             conda-forge/noarch::snuggs-1.4.7-py_0
  sphinx             conda-forge/noarch::sphinx-3.1.1-py_0
  sphinx_rtd_theme   conda-forge/noarch::sphinx_rtd_theme-0.5.0-pyh9f0ad1d_0
  sphinxcontrib-app~ conda-forge/noarch::sphinxcontrib-applehelp-1.0.2-py_0
  sphinxcontrib-dev~ conda-forge/noarch::sphinxcontrib-devhelp-1.0.2-py_0
  sphinxcontrib-htm~ conda-forge/noarch::sphinxcontrib-htmlhelp-1.0.3-py_0
  sphinxcontrib-jsm~ conda-forge/noarch::sphinxcontrib-jsmath-1.0.1-py_0
  sphinxcontrib-qth~ conda-forge/noarch::sphinxcontrib-qthelp-1.0.3-py_0
  sphinxcontrib-ser~ conda-forge/noarch::sphinxcontrib-serializinghtml-1.1.4-py_0
  tbb                conda-forge/osx-64::tbb-2019.9-ha1b3eb9_1
  tiledb             conda-forge/osx-64::tiledb-1.7.7-h84aa2a7_3
  tornado            conda-forge/osx-64::tornado-6.0.4-py37h9bfed18_1
  tqdm               conda-forge/noarch::tqdm-4.46.1-pyh9f0ad1d_0
  twine              conda-forge/osx-64::twine-3.1.1-py37_0
  tzcode             conda-forge/osx-64::tzcode-2020a-h0b31af3_0
  urllib3            conda-forge/noarch::urllib3-1.25.9-py_0
  wcwidth            conda-forge/noarch::wcwidth-0.2.4-pyh9f0ad1d_0
  webencodings       conda-forge/noarch::webencodings-0.5.1-py_1
  xerces-c           conda-forge/osx-64::xerces-c-3.2.2-h8f8adb3_1004
  xmltodict          conda-forge/noarch::xmltodict-0.12.0-py_0
  yaml               conda-forge/osx-64::yaml-0.2.5-h0b31af3_0
  zipp               conda-forge/noarch::zipp-3.1.0-py_0
  zstd               conda-forge/osx-64::zstd-1.4.4-h4b3e974_3


Proceed ([y]/n)? y


Downloading and Extracting Packages
numpy-1.18.5         | 5.0 MB    | ################################################################################################################################################################# | 100%
recommonmark-0.6.0   | 13 KB     | ################################################################################################################################################################# | 100%
simplejson-3.17.0    | 100 KB    | ################################################################################################################################################################# | 100%
sphinx_rtd_theme-0.5 | 2.7 MB    | ################################################################################################################################################################# | 100%
xmltodict-0.12.0     | 11 KB     | ################################################################################################################################################################# | 100%
deepforest-0.2.19    | 8.8 MB    | ################################################################################################################################################################# | 100%
numpydoc-1.0.0       | 41 KB     | ################################################################################################################################################################# | 100%
gdal-3.0.4           | 1.4 MB    | ################################################################################################################################################################# | 100%
sphinx-3.1.1         | 1.4 MB    | ################################################################################################################################################################# | 100%
keyring-21.2.1       | 46 KB     | ################################################################################################################################################################# | 100%
rasterstats-0.14.0   | 16 KB     | ################################################################################################################################################################# | 100%
libgdal-3.0.4        | 17.2 MB   | ################################################################################################################################################################# | 100%
pandas-1.0.5         | 9.6 MB    | ################################################################################################################################################################# | 100%
requests-2.24.0      | 47 KB     | ################################################################################################################################################################# | 100%
twine-3.1.1          | 47 KB     | ################################################################################################################################################################# | 100%
cython-0.29.20       | 2.0 MB    | ################################################################################################################################################################# | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
(test) MacBook-Pro:Figures ben$ python
Python 3.7.6 | packaged by conda-forge | (default, Jun  1 2020, 18:33:30)
[Clang 9.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import deepforest
>>> from deepforest import deepforest
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/ben/miniconda3/envs/test/lib/python3.7/site-packages/deepforest/deepforest.py", line 15, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'
```

**Comment by bw4sz** [comment]
https://github.com/conda-forge/deepforest-feedstock/blob/ca2bad7be5ab5b7f520c913b9e8ec358e8af5d76/recipe/meta.yaml#L55

the requirements.txt is not present in this repo?

**Comment by bw4sz** [comment]
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=176209&view=logs&j=517fe804-fa30-5dc2-1413-330699242c05&t=c7d0fcb4-3a10-5b26-dfea-e879a93fc2ab&l=861 no tests are build run, see 

```
Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
```

**Comment by bw4sz** [comment]
Solved by #3
